contracts/wallet/WalletFactory.sol

```diff
@@ -109,10 +106,7 @@ contract WalletFactory is Owned, Managed {
         onlyManager
         guardianStorageDefined
     {
-        validateInputs(_owner, _modules, _label);
-        Proxy proxy = new Proxy(walletImplementation);
-        address payable wallet = address(proxy);
-        configureWallet(BaseWallet(wallet), _owner, _modules, _label, _guardian);
+        _createWallet(_owner, _modules, _label, _guardian);
```

Comment:
We should validate the guardian address is non empty here as it's not done further down the call chain

Reply:
Fixed as well.